src/PrivacyProtection/PrivacyProtection.jsx

```diff
@@ -115,7 +115,7 @@ const PrivacyProtection = (props) => {
 
   const height = React.useMemo(() => {
     if (!props.height) return 'auto';
-    if (isNumber(parseInt(props.height))) return `${props.height}px`;
```

Comment:
@nileshgulia1 isNumber doesn't seems to handle string numbers like:

<pre>
isNumber("33.33");
</pre>

Thus, I think parseInt was added there with a reason. IMHO, you should rather add a try catch if it fails .

Reply:
@avoinea see: [f7dd30c](https://github.com/eea/volto-embed/pull/63/commits/f7dd30c47aa2cc31726e49b8eb4cb40aae624941)